utils/estimated_histogram.hh

```diff
@@ -192,31 +216,15 @@ public:
      * @return the smallest value that could have been added to this histogram
      */
     int64_t min() const {
-        size_t i = 0;
-        for (auto b : buckets) {
-            if (b > 0) {
-                return i == 0 ? 0 : 1 + bucket_offsets[i - 1];
-            }
-            i++;
-        }
-        return 0;
+        return _min;
```

Comment:
What's the motivation? we get better min/max, but also do more work per operation. Is there a problem with the current min/max?

Reply:
it's not a critical problem, but there is no other way of knowing what is the real min and max.